examples/example.ipynb

```
@@ -0,0 +1,430 @@
+                "        default_related_name = \"book_copies\"\n",
+                "        verbose_name_plural = \"Book copies\"\n",
+                "\n",
+                "\n",
+                "class Borrowing(models.Model):\n",
+                "    user = models.ForeignKey(User, on_delete=models.CASCADE)  # Foreign Key relationship\n",
+                "    book_copy = models.ForeignKey(BookCopy, on_delete=models.CASCADE)  # Foreign Key relationship\n",
+                "    borrow_date = models.DateField()\n",
+                "    return_date = models.DateField(null=True, blank=True)\n",
+                "\n",
+                "    class Meta:\n",
+                "        app_label = \"tests\"\n",
```

Ruff (0.11.9)

Module level import not at top of file

(E402)